DEVELOPING.md

```diff
@@ -54,6 +53,14 @@ To debug the LemMinX Language Server in IntelliJ, complete the following steps.
 1. Follow the Eclipse LSP4MP [build instructions](https://github.com/eclipse/lsp4mp#getting-started) to build the `org.eclipse.lsp4mp.ls-x.x.x-SNAPSHOT-jar-with-dependencies.jar`. You do not need to build the `lsp4mp.jdt` component.
 2. In the [build.gradle file](build.gradle), update references to the `org.eclipse.lsp4mp.ls` to point to the version built in the previous step.
 
+### Monitoring Language Server Messages
```

Comment:
```suggestion
### Monitoring Language Server messages
```

Reply:
As long as we're talking, shouldn't we use title caps here? Or maybe Language and Server could be lower case too?

Reply:
Yes- I noticed that in the unedited section of the doc "Lanaguage Server" is capitalized in the titles, so I left it capitalized here for consistency. But probably more correct to change this and the other instances to lowercase ( except "Language servers")